modules/monitoring-setting-up-pod-topology-spread-constraints-for-thanos-ruler.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-up-pod-topology-spread-constraints-for-thanos-ruler_{context}"]
+= Setting up pod topology spread constraints for Thanos Ruler
+
+For user-defined monitoring, you can set up pod topology spread constraints for Thanos Ruler to fine tune how pod replicas are scheduled to nodes across zones. 
+Doing so helps ensure that Thanos Ruler pods are highly available and run more efficiently because workloads are spread across nodes in different data centers or hierarchical infrastructure zones.
+
+You configure topology spread constraints for user-defined monitoring in the `cluster-monitoring-config` config map.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
```

Comment:
add
A cluster administrator has enabled monitoring for user-defined projects

Reply:
Added